src/linux-release-info.ts

```diff
@@ -0,0 +1,218 @@
+/*!
+ * linux-release-info
```

Comment:
Can you drop a permalink in here of where exactly we got this from? owner/repo / commit / filename?